app/routes/events/components/EventDetail/index.js

```diff
@@ -210,35 +212,56 @@ export default class EventDetail extends Component<Props> {
                   <InfoList items={paidItems} />
                 </div>
               )}
+              <Flex column>
+                <h3>Påmeldte</h3>
+                {registrations ? (
+                  [
+                    <Flex
+                      key="registrations"
+                      className={styles.registeredThumbnails}
+                    >
+                      {registrations
+                        .slice(0, 12)
+                        .map(reg => (
+                          <RegisteredCell key={reg.user.id} user={reg.user} />
+                        ))}
+                    </Flex>,
+                    <ModalParentComponent
+                      key="modal"
+                      pools={pools}
+                      title="Påmeldte"
+                    >
+                      <RegisteredSummary registrations={registrations} />
+                      <AttendanceStatus />
+                    </ModalParentComponent>
+                  ]
+                ) : (
+                  <AttendanceStatus isCount useModal={false} pools={pools} />
+                )}
 
-              {loggedIn && (
-                <Flex column>
-                  <h3>Påmeldte</h3>
-                  <Flex className={styles.registeredThumbnails}>
-                    {registrations
-                      .slice(0, 12)
-                      .map(reg => (
-                        <RegisteredCell key={reg.user.id} user={reg.user} />
-                      ))}
-                  </Flex>
-                  <ModalParentComponent pools={pools} title="Påmeldte">
-                    <RegisteredSummary registrations={registrations} />
-                    <AttendanceStatus />
-                  </ModalParentComponent>
-
+                {loggedIn && (
                   <RegistrationMeta
                     registration={currentRegistration}
                     isPriced={event.isPriced}
                   />
-                  <Admin
-                    actionGrant={actionGrant}
-                    event={event}
-                    deleteEvent={deleteEvent}
-                  />
-                </Flex>
-              )}
+                )}
+                <Admin
+                  actionGrant={actionGrant}
+                  event={event}
+                  deleteEvent={deleteEvent}
+                />
+              </Flex>
             </ContentSidebar>
           </ContentSection>
+          <div className={styles.joinHeader}>
+            Bli med på dette arrangementet
```

Comment:
A link for users to log in to register would be nice! :)

Reply:
:+1:

Just need a nice way to do it

Reply:
Any idea about how we can do it? Should we just create a custom route with redirect back to the event? An inline login form (visible after eg. a click) will look kinda shady imo.

Reply:
Not sure what the best way is, but I think the behaviour I'd expect would be a simple link "Click here to login", which would take me to a login page that directs me back to the event after logging in.